test/response_test.rb

```diff
@@ -1,5 +1,7 @@
 require 'xsr'
 require_relative 'helper'
+require 'codeclimate-test-reporter'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.